docs/configuration/upstream/propose_downstream.md

```diff
@@ -5,8 +5,15 @@ sidebar_position: 5
 
 # `job: propose_downstream`
 
-Land a new upstream release in Fedora. This job only makes sure the changes
-happen in Fedora dist-git - no builds. A pull request is created as a result.
+Land a new upstream release in Fedora (or CentOS Stream). This job only makes sure the changes
+happen in dist-git - no builds. A pull request is created as a result.
+
+:::tip CentOS Stream support
+
+We newly support `propose-downstream` also for CentOS Stream, see the configuration specifics in the 
+[subsection below](#syncing-the-release-to-centos-stream).
```

Comment:
```suggestion
We newly support `propose-downstream` also for CentOS Stream. The workflow is the same as for Fedora, but see the configuration specifics in the 
[subsection below](#syncing-the-release-to-centos-stream).
```

Or something like that to let people know that the process of the sync itself is the same.